codersdk/experiments.go

```diff
@@ -0,0 +1,48 @@
+package codersdk
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+)
+
+var (
+	// ExperimentVSCodeLocal enables a workspace button to launch VSCode
+	// and connect using the local VSCode extension.
+	ExperimentVSCodeLocal = "vscode_local"
+	// ExperimentsAll should include all experiments that are safe for
+	// users to opt-in to via --experimental='*'.
+	// Experiments that are not ready for consumption by all users should
+	// not be included here.
+	ExperimentsAll Experiments = Experiments{
```

Comment:
Suggestion: Simplify

```suggestion
	ExperimentsAll = Experiments{
```